sig-architecture/annual-report-2021.md

```diff
@@ -87,51 +147,84 @@ In future, this will be generated from delta of sigs.yaml from $YYYY-01-01 to $Y
+    - At the start of 2021 the tracking of ineligible endpoints for conformance was moved to
+      [`ineligible_endpoints.yaml`](https://github.com/kubernetes/kubernetes/blob/master/test/conformance/testdata/ineligible_endpoints.yaml) file.
+      This allows for more community engagement on the eligibility of endpoints for conformance before any endpoint is added to the endpoint to the 
+      [Ineligible Endpoints list](https://apisnoop.cncf.io/conformance-progress/ineligible-endpoints) through pubic pull request.
+    - Our target for 2022 is to clean up the last 51 endpoint and then move over to Conformance profiles 
+      to further increase the value added by conformance testing.
+- [Code Organization](https://github.com/kubernetes/community/tree/master/sig-architecture#code-organization-1)
+     
+- [Enhancements](https://github.com/kubernetes/community/tree/master/sig-architecture#enhancement-proposals)
+    - In 2021, the subproject was mainly focused on improving the contributor experience via automation and thoughtful ui/ux changes making visible changes to the KEP process.
+    - Major efforts were put into improving tooling on the enhancements repo throughout the year allowing us to automate more validation lessening the burden on individual approvers and resulting in more accurate information reflected in the document. All KEPs were migrated to the new template which now includes a key.yaml file.
+    - The team had major contributions authoring and negotiating [KEP 2572 - Defining the Kubernetes release cadence](https://github.com/kubernetes/enhancements/blob/master/keps/sig-release/2572-release-cadence/).
+    - We made progress on the receipts process, but ultimately decided that it was too disruptive for the community as a new UI/UX.
+    - The team updated the repository documentation and work was started on a KEP website for easier navigation.
+    - New members joined the team: 1 new approver and 2 reviewers were added to the Enhancements Subproject. A new kep tools team with initially 4 reviewers and approvers was created and more recently had an addition of 1 new approver/reviewer. We also had 2 owners transition to become our first Emeritus approvers.
+    - We hope to keep improving the KEP process in 2022 and will be conducting a community survey to collect feedback, working on a Process KEP template and other improvements.
+- [Production Readiness](https://github.com/kubernetes/community/tree/master/sig-architecture#production-readiness-1)
+    - Production readiness review [became mandatory] for enhancements in 2021, starting with the 1.21 release cycle.
+    - We added a fourth approver, Elana Hashman, who joined the team after shadowing in 1.21.
+    - We created documentation on the [lifecycle of feature gates].
+    - We reviewed data from the 2020 Production Readiness Survey, and prepared a 2021 survey, whose [responses we analyzed]. # **TODO:** @dims to update link to point to YouTube recording
```

Comment:
TODO for @dims

Reply:
Survey report URL: https://datastudio.google.com/reporting/2e9c7439-202b-48a9-8c57-4459e0d69c8d/page/GC5HB

Reply:
YouTube: https://youtu.be/9GM28aAyRQs

I was WAAAAAYYYYY behind on uploading....

Reply:
updating the report with the 2 urls